### ISSUE TITLE
Playlists with Relative paths broken 

### ISSUE THREAD

**Issue by erkr** [music-assistant/hass-music-assistant#712]
### What version of Music Assistant has the issue?

2022.7.0

### The problem

I have Playlists without paths in the same folders as my local media files. These are all empty now in MA. 
Not sure when this has changed. Was working before the betas.

### How to reproduce

Open a Playlist with only file names that is located in the same folder as the media files 

### Relevant log output

Logger: music_assistant.music.file
Source: /usr/local/lib/python3.10/site-packages/music_assistant/music_providers/filesystem.py:382 
First occurred: 17:53:16 (472 occurrences) 
Last logged: 17:54:51

Could not parse uri/file 06 - You're My Temptation.mp3 to track: Not a valid Music Assistant uri: 06 - You're My Temptation.mp3
Could not parse uri/file 07 - Trash.mp3 to track: Not a valid Music Assistant uri: 07 - Trash.mp3
Could not parse uri/file 08 - Stolen Prayer.mp3 to track: Not a valid Music Assistant uri: 08 - Stolen Prayer.mp3
Could not parse uri/file 09 - Sideshow.mp3 to track: Not a valid Music Assistant uri: 09 - Sideshow.mp3
Could not parse uri/file 10 - Might As Well Be On Mars.mp3 to track: Not a valid Music Assistant uri: 10 - Might As Well Be On Mars.mp3

### What version of Home Assistant Core are your running

Core-2022.6.7 

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Raspberry Pi

**Comment by marcelveldt** [comment]
should be fixed in 2022.7.1 but I did not test it, can you give it a try ?

**Comment by erkr** [comment]
No still empty. Same kind loggings:
``` 
Logger: music_assistant.music.file
Source: /usr/local/lib/python3.10/site-packages/music_assistant/music_providers/filesystem.py:401 
First occurred: 10:53:22 (90 occurrences) 
Last logged: 10:53:37

Could not parse uri/file 56 - 14 - Canadian sunset - Hugo Winterhalter.MP3 to track: Not a valid Music Assistant uri: 56 - 14 - Canadian sunset - Hugo Winterhalter.MP3
Could not parse uri/file 56 - 15 - When my dreamboat comes home - Fats Domino.MP3 to track: Not a valid Music Assistant uri: 56 - 15 - When my dreamboat comes home - Fats Domino.MP3
Could not parse uri/file 56 - 16 - Cindy,Oh Cindy - Eddie Fisher.MP3 to track: Not a valid Music Assistant uri: 56 - 16 - Cindy,Oh Cindy - Eddie Fisher.MP3
Could not parse uri/file 56 - 17 - Zambesi - Lou Bush Orchestra.MP3 to track: Not a valid Music Assistant uri: 56 - 17 - Zambesi - Lou Bush Orchestra.MP3
Could not parse uri/file 56 - 18 - No other love - Ronnie Hilton.MP3 to track: Not a valid Music Assistant uri: 56 - 18 - No other love - Ronnie Hilton.MP3
```

**Comment by marcelveldt** [comment]
Fixed in 2022.7.2 - tested myself this time

playlists can now contain:
- paths relative to the musicdir
- paths relative to the playlist
- musicprovider uri's
- url's
